llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py

```diff
@@ -329,6 +310,7 @@ def from_documents(  # type: ignore
                 for doc in documents
             ],
         )
+
         doc_ids = [doc.id for doc in upserted_documents]
         index._wait_for_documents_ingestion(
             doc_ids, verbose=verbose, raise_on_error=raise_on_error
```

Comment:
Is the URL printed below still accurate? `print(f"Find your index at {app_url}/project/{project.id}/deploy/{pipeline.id}")`

Reply:
yes